products/developments/housing_post2010/metadata.yml

```diff
@@ -0,0 +1,548 @@
+    description: Geometry is created from latitude and longitude provided in the manual corrections file"
+- id: dcpedited
+  name: DCPEdited
+  data_type: text
+  description: >
+    List of fields that were edited by DCP. This only lists fields where original DOB source data was overwritten by DCP, 
+    and doesn't include fields where DCP recodes data as part of the standard methodology.
+  example: classa_init
+- id: version
+  name: Version
+  data_type: text
+  description: Version of the Developments Database.
+  example: 23Q4
+- id: location
+  name: Location
+  data_type: geometry
+  example: '{''coordinates'': [-73.920731, 40.556881], ''type'': ''Point''}'
+- id: bldg_class
+  name: Bldg_Class
+  data_type: text
+  description: Building Class code. A code used by Department of Finance (DOF) and Department of Buildings (DOB) describing the major use of a structure.
+  example: A1
+- id: centract10
+  name: CenTract10
+  data_type: text
+  description: 2010 Census Tract FIPS code.
+  example: '36081091603'
+- id: cenblock20
+  name: CenBlock20
+  data_type: text
+  description: 2020 Census Block FIPS code.
+  example: '360471070011011'
+- id: centract20
+  name: CenTract20
+  data_type: text
+  description: 2020 Census Tract FIPS code.
+  example: '36081091603'
+- id: bctcb2020
+  name: BCTCB2020
+  data_type: text
+  description: The Borough - 2020 Census Tract - Census Block code.
+  example: '31070011011'
+- id: bct2020
+  name: BCT2020
+  data_type: text
+  description: The Borough - 2020 Census Tract code.
+  example: '4091603'
+- id: nta2020
+  name: NTA2020
+  data_type: text
+  description: 2020 Neighborhood Tabulation Area (NTA) code.
+  example: SI0304
+- id: ntaname20
+  name: NTAName20
+  data_type: text
+  description: 2020 Neighborhood Tabulation Area (NTA) descriptive name.
+  example: Annadale-Huguenot-Prince's Bay-Woodrow
+- id: cdta2020
+  name: CDTA2020
+  data_type: text
+  description: 2020 Community District Tabulation Area (NTA) code.
+  example: SI03
+- id: cdtaname20
+  name: CDTAName20
+  data_type: text
+  description: 2020 Community District Tabulation Area (NTA) descriptive name.
+  example: SI03 South Shore (CD 3 Approximation)
+- id: :@computed_region_efsh_h5xi
+  name: Zip Codes
+  data_type: integer
+  description: This column was automatically created in order to record in what polygon
+    from the dataset 'Zip Codes' (efsh-h5xi) the point in column 'location' is located.  This
+    enables the creation of region maps (choropleths) in the visualization canvas
+    and data lens.
+- id: :@computed_region_f5dn_yrer
+  name: Community Districts
+  data_type: text
+  description: This column was automatically created in order to record in what polygon
+    from the dataset 'Community Districts' (f5dn-yrer) the point in column 'location'
+    is located.  This enables the creation of region maps (choropleths) in the visualization
+    canvas and data lens.
+- id: :@computed_region_yeji_bk3q
```

Comment:
These are columns that are from other datasets in socrata? Curious if you've run into this @alexrichey

Reply:
this is a strange case. I have seen computed columns in the past, and I've ignored them in the metadata. You should be able to remove them here and still push. (I believe those computed columns persist, but... if they don't, I'm not sure that's necessarily a bad thing.) I'll check with GIS about that.